src/convexhull.jl

```diff
@@ -0,0 +1,36 @@
+"""
+    upperhull(pset)
+
+Upper part of the convex hull of a given set of 2D points `pset`. 
+The upperhull is computed by the "Graham Scan" algorithm.
+"""
+function upperhull(pset::PointSet)
+
+  xs=sort(coordinates.(pset), by=first)
+  
+  upperHull=[xs[1],xs[2]] 
```

Comment:
Please rewrite all assignments with whitespaces `a = 1` between left and right hand side.